data-prepper-plugins/aggregate-processor/src/main/java/com/amazon/dataprepper/plugins/processor/aggregate/AggregateActionSynchronizer.java

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Could you run the code formatter on these files? I noticed a few minor formatting inconsistencies that IntelliJ should be able to clean up.

Reply:
Which code formatter? ‘spotlessApply’?

Reply:
I do think through IntelliJ. It is under _Code_ -> _Reformat Code_.